stl/inc/vector

```diff
@@ -539,33 +539,10 @@ private:
         _Apply_annotation(_My_data._Myfirst, _My_data._Myend, _My_data._Mylast, _My_data._Mylast + _Count);
     }
 
-    _NODISCARD static const void* _Get_aligned_first(const void* _First, const void* _End) noexcept {
-        const auto _CFirst     = reinterpret_cast<const char*>(_First);
-        const auto _CEnd       = reinterpret_cast<const char*>(_End);
-        const size_t _Capacity = static_cast<size_t>(_CEnd - _CFirst);
-
-        if (_Capacity >= _Asan_granularity) {
-            // We are guaranteed to have sufficient space to find an aligned address.
-            return reinterpret_cast<const void*>(
-                (reinterpret_cast<uintptr_t>(_CFirst) + (_Asan_granularity - 1)) & ~(_Asan_granularity - 1));
-        }
-
-        uintptr_t _Alignment_offset = reinterpret_cast<uintptr_t>(_CFirst) & (_Asan_granularity - 1);
-        if (_Alignment_offset != 0) {
-            _Alignment_offset = _Asan_granularity - _Alignment_offset;
-        }
-
-        if (_Capacity > _Alignment_offset) {
-            return _CFirst + _Alignment_offset;
-        }
-
-        return nullptr;
-    }
-
     static _CONSTEXPR20 void _Apply_annotation(
-        pointer _First_, pointer _End_, pointer _Old_last_, pointer _New_last_) noexcept {
+        pointer _First_, pointer _Buffer_last_, pointer _Old_last_, pointer _New_last_) noexcept {

```

Comment:
(Sung to the tune of "you kids get off my lawn!") Why are we renaming `_End`? We have a lot of history with using `first`, `last`, and `end` to refer to the three pointers in a `vector`.

Reply:
I... really dislike this convention, but I'll follow it. `_End` confused the heck out of me.